pkg/storage/segments/repairer_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package segments_test
+
+import (
+	"crypto/rand"
+	"testing"
+	time "time"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pb"
+	ecclient "storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storage/segments"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestSegmentStoreRepair(t *testing.T) {
+	numStorageNodes := 10
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: numStorageNodes, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		// first, upload some remote data
+		uplink := planet.Uplinks[0]
+		satellite := planet.Satellites[0]
+
+		testData := make([]byte, 5*memory.MiB)
+		_, err := rand.Read(testData)
+		assert.NoError(t, err)
+
+		err = uplink.Upload(ctx, satellite, "test/bucket", "test/path", testData)
+		assert.NoError(t, err)
```

Comment:
`require.NoError(t, err)`, since the test following it doesn't make sense if it fails.